### ISSUE TITLE
[iOS] My edited photo is reverted from the origin photo after uploading to server

### ISSUE THREAD

**Issue by dao-wkm** [Sh1d0w/multi_image_picker#277]
**Bug Report**

`multi_image_picker` version: 4.5.6

Is it happening on Android: No

Is it happening on iOS: Yes (real device iOS 12.1.4)

```
[✓] Flutter (Channel stable, v1.9.1+hotfix.2, on Mac OS X 10.14.6 18G95, locale en-VN)
 
[✓] Android toolchain - develop for Android devices (Android SDK version 28.0.3)
[✓] Xcode - develop for iOS and macOS (Xcode 11.0)
[✓] Android Studio (version 3.4)
[✓] VS Code (version 1.38.1)
[✓] Connected device (2 available)
```

**Describe the bug**
When i use iPhone Photos Editor, my edited photo is reverted from the origin photo after uploading to server.

*My Code:*
```
import 'dart:typed_data';
import 'package:path/path.dart';
import 'package:dio/dio.dart';

Future uploadImage(List<Asset> images) async {
 ...
 for (Asset image in images) {
    String imageName = basename(file.path);
    ByteData byteData = await image.getByteData(quality: 90);
    Uint8List uint8List = byteData.buffer.asUint8List();
    UploadFileInfo.fromBytes(uint8List, imageName);
 }
 ...
}
```

*Before: *
![Before](https://user-images.githubusercontent.com/42726313/66450095-75d3b400-ea81-11e9-827d-6ab8af712345.PNG)

*After:*
![After](https://user-images.githubusercontent.com/42726313/66450122-8a17b100-ea81-11e9-9ee9-6489e0b50ffe.PNG)

**Comment by Sh1d0w** [comment]
Hi, this should be fixed in version 4.5.8

**Comment by dao-wkm** [comment]
> Hi, this should be fixed in version 4.5.8

thanks
